### ISSUE TITLE
Helpfile is not correct?

### ISSUE THREAD

**Issue by nutzer0927** [swillner/highlight-pointer#12]
First of all: A big **Thank You** for this great tiny programme :)

I encountered the following problem:
The helpfile says:
--released-color COLOR  dot color when mouse button released [default: #d62728]

so I tried a different colour with
--released-color #9acd32
which did not work

Only after reading slavas comment in
https://github.com/swillner/highlight-pointer/issues/3

I realized that the color-code hat to be set between '...'
--released-color '#9acd32'

So maybe you should hint out in the helpfile that the colour-code ist to be set between '...'

**Comment by hussain1406** [comment]
yes and for anyone wondering why? it is because # is treated as a comment in bash so everything written after # will be ignored by bash and not passed to the program itself so definitely use either '...' single quote "..." or a double quote around the hexadecimal code

**Comment by swillner** [comment]
good spot @ghost, thanks! will adjust the helpfile ;)
